catroid/src/org/catrobat/catroid/pocketmusic/note/trackgrid/GridRow.java

```diff
@@ -52,7 +51,7 @@ public int hashCode() {
 		int result = 21;
 		result = 31 * result + noteName.hashCode();
```

Comment:
Why 31? Can you use a variable for 31 for a better understanding of the code? result isn't a good name.. which result do you mean?